apps/web-giddh/src/app/gst/filing/filing.component.ts

```diff
@@ -91,7 +91,7 @@ export class FilingComponent implements OnInit, OnDestroy {
     public ngOnInit() {
         document.querySelector('body').classList.add('gst-sidebar-open');
         this.breakpointObserver
-        .observe(['(max-width: 767px)'])
+        .observe(['(max-width: 768px)'])
```

Comment:
@shalinee-walkover why we are using 768px in place of 767px? ideal media query point is 767px.

Reply:
For the ipad view , 768px is ideal resolution.